nss-day-cohort-30/bangazon-api-maniacal-magpies
# Order controller bn

# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Order Controller and Order Testing completed and DONE according to Bangazon standards.

## Type of change
Please delete options that are not relevant.

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout OrderController-BN`
3. run full database reset along with insert statement of verified group test data
4. run program...
5. verify all tests should pass


# Checklist:
- [ X] My code follows the style guidelines of this project
- [ X] I have performed a self-review of my own code
- [ X] I have commented my code, particularly in hard-to-understand areas
- [ X] My changes generate no new warnings
- [ X] I have added test instructions that prove my fix is effective or that my feature works
